Python/Product/PythonTools/PythonTools/Refactoring/ExtractMethodRequestView.cs

```diff
@@ -256,9 +257,14 @@ public ExtractMethodRequestView(IServiceProvider serviceProvider, ExtractedMetho
         private void UpdatePreview() {
             var info = GetRequest();
             if (info != null) {
-                _previewer.GetExtractionResult(info).ContinueWith(
-                    x => PreviewText = x?.WaitOrDefault(1000)?.methodBody ?? Strings.ExtractMethod_FailedToGetPreview
-                ).DoNotWait();
+                _previewer.GetExtractionResult(info).ContinueWith(t => {
+                    try {
+                        PreviewText = t.Result.methodBody;
```

Comment:
I found what I was thinking of, and it wasn't this.
https://github.com/Microsoft/PTVS/pull/2161/files